### PR TITLE
CloudFM to CloudX + API Integration

### PR DESCRIPTION
**Pull request by marinofranz**

### CODE REVIEW

**Comment by marinofranz** [comment]
image
image

File: websites/C/CloudX/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "author": {
+    "name": "Marino",
+    "id": "598953747963707409"
+  },
+  "url": "thisiscloudx.com",
+  "description": {
+    "en": "CloudX is an online radio station that strives to provide you with the best and latest hit from all around the world!"
+  },
+  "service": "CloudX",
+  "version": "1.0.1",
```
Comment (Bas950) [inline websites/C/CloudX/dist/metadata.json:12]: version 1.0.0 on first release

**Review by Bas950 — CHANGES_REQUESTED** [review]
Dont upload js file, fix deepscan issues

File: websites/C/CloudX/dist/metadata.json (hunk 1 above)
Comment (marinofranz) [reply websites/C/CloudX/dist/metadata.json]: fixed

**Review by Bas950 — APPROVED** [review]

**Pull request merged**